deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -167,42 +168,6 @@ def default_generator(self,
         feed_dict[self.atom_to_pair] = np.concatenate(atom_to_pair, axis=0)
         yield feed_dict
 
-  def predict(self, dataset, transformers=[], batch_size=None):
```

Comment:
I have double checked, these are exactly the same functions used in the main `TensorGraph` . I added them first because tensorgraph doesn't support these graph models at that time. All test run are normal too.